Formula/lima.rb

```diff
@@ -0,0 +1,29 @@
+class Lima < Formula
+  desc "Linux virtual machines"
+  homepage "https://github.com/AkihiroSuda/lima"
+  url "https://github.com/AkihiroSuda/lima.git",
+    tag:      "v0.3.0",
+    revision: "c82737b29cf54225c3ec8ec7073609d42924017c"
```

Comment:
```suggestion
      tag:      "v0.3.0",
      revision: "c82737b29cf54225c3ec8ec7073609d42924017c"
```
Let's align the arguments to `url`.

Reply:
Thx

Reply:
I changed to source to the `tar` file and add the `VERSION` option.